docs/sources/plugins/developing/datasources.md

```diff
@@ -146,7 +146,8 @@ Request object passed to datasource.annotationQuery function:
     "datasource": "generic datasource",
     "enable": true,
     "name": "annotation name"
-  }
+  },
+  "dashboard": DashboardModel
```

Comment:
This is also passed on but not sure if we wan't to enumerate everything here. Tbh I would rather put some link to our sources with the particular type to not duplicate it at all.